torchbraid/odenet_apps.py

```diff
@@ -141,6 +141,9 @@ def updateParallelWeights(self):
   def run(self,x):
     # turn on derivative path (as requried)
     self.use_deriv = self.training
+    
+    # instead of doing runBraid, can execute tests
+    #self.testBraid(x)
```

Comment:
Thanks for adding this, didn't know about it.

Reply:
I added these for David -- they aren't terribly useful, but they do ensure some basic functionality.